File: xh_manager/src/main/java/com/xh/service/BaseCodeService.java
package com.xh.service;

import java.util.List;

import com.xh.pojo.BaseCode;

public interface BaseCodeService {
	
	public List<BaseCode> findBaseCodeByTypeCode(String typeCode);
	
	public List<BaseCode> findBaseCodeByTypeCodeList(List<String> typeCodeList);
	
	public List<BaseCode> findAll();
}

File: xh_manager/src/main/webapp/js/service/baseCodeService.js
app.service("baseCodeService",function($http){
	
	//查询所有的基础选项数据
	this.findBaseCodeList = function(){
		return $http.get('../baseCode/findAllBaseCodeList.action');
	};
	
	//查询所有的基础选项数据
	this.findBaseCodeByTypeCode = function(typeCode){
		return $http.get('../baseCode/findBaseCodeByTypeCode.action?typeCode='+typeCode);
	};
	
	//查询所有
	this.findAll = function(){
		return $http.get('../baseCode/findAll.action');
	};
	
});